src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -494,22 +545,30 @@ public void producesPartsWithContentLength() throws IOException {
                 ),
                 Joiner.on("\r\n").join(
                     "--AaB01x",
-                    "Content-Disposition: form-data; name=\"t2\"",
+                    new StringBuilder()
```

Comment:
after the changes the word "t2" appeared twice in the code, so I need to create a constant, and concat with Strings are not possible, so I used the StringBuilder to do that.